src/utils/loadable.jsx

```diff
@@ -0,0 +1,38 @@
+import { Fragment, PureComponent } from 'react';
+import Loadable from 'react-loadable';
+import { withStyles } from '@material-ui/core/styles';
+import Spinner from '../components/Spinner';
+
+@withStyles(theme => ({
+  view: {
+    textAlign: 'center',
+    margin: 100,
+    backgroundColor: theme.palette.background.default,
+    height: 'calc(100% - 60px)',
+    marginTop: 60,
+  },
+}))
+class Loading extends PureComponent {
+  content() {
+    const { classes, error, timedOut, pastDelay } = this.props;
+
+    if (error) {
+      throw error;
+    } else if (timedOut || pastDelay) {
+      return <Spinner className={classes.view} />;
```

Comment:
I would have the parent be a `div` and let it position the spinner. In other words, the render method should be `<div className={classes.view}>{this.content()}</div>`.